app/src/main/java/com/internship/move/network/model/Requests.kt

```diff
@@ -0,0 +1,17 @@
+package com.internship.move.network.model
```

Comment:
For these requests/response files:
I suggest creating feature sub-packages that hold their respective DTO classes. (Also, you could rename the `model` package to `dto` in the `network` package, but that's entirely subjective)